checker.go

```diff
@@ -6,6 +6,14 @@ import (
 	"strings"
 )
 
+type sign int
```

Comment:
You're right. It may be considered as FlagArgument. Any better idea how not to repeat the whole `calculateScore` body. I'm thinking of callback function but seems like an overkill in this case.